contrib/firewalld/patches/0001-don-t-build-tests.patch

```diff
@@ -0,0 +1,188 @@
+From fbdfd3fd6a00fef6342550d63ae090c2daa360db Mon Sep 17 00:00:00 2001
+From: miko <mikoxyzzz@gmail.com>
+Date: Sat, 13 Jan 2024 19:23:23 +0100
+Subject: [PATCH] don't "build" tests
+
+build breaks if we don't do this. the tests don't even work in our
+environment anyway
+---
+ Makefile.am           |   4 --
+ src/tests/Makefile.am | 151 ------------------------------------------
+ 2 files changed, 155 deletions(-)
+ delete mode 100644 src/tests/Makefile.am
+
+diff --git a/Makefile.am b/Makefile.am
+index 8e293653..1e762939 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -26,10 +26,6 @@ dist-container:
+ 	$(PODMAN) push quay.io/firewalld/firewalld:latest
+ 	$(PODMAN) save --format oci-archive --output firewalld-oci-$(PACKAGE_VERSION).tar firewalld:$(PACKAGE_VERSION)
+ 
+-check-container check-integration installcheck-integration:
+-	$(MAKE) -C src/tests $@
+-
+-.PHONY: check-container check-integration installcheck-integration
+ .PHONY: dist-container
+ 
+ rpm test-rpm: dist
+diff --git a/src/tests/Makefile.am b/src/tests/Makefile.am
+deleted file mode 100644
+index 62b88d61..00000000
+--- a/src/tests/Makefile.am
```

Comment:
do you actually need to delete the whole file? normally you just have to remove some SUBDIR from a parent .am to ignore it instead

Reply:
same for below, rather than a 5k line patch file you probably need 2 lines

Reply:
> do you actually need to delete the whole file? normally you just have to remove some SUBDIR from a parent .am to ignore it instead

fuck im so silly